tests/runner/auction/output_3.json

```diff
@@ -2,12 +2,14 @@
   "scilla_major_version": "0",
   "gas_remaining": "7423",
   "_accepted": "false",
-  "message": {
-    "_tag": "",
-    "_amount": "0",
-    "_recipient": "0x12345678901234567890123456789012345678cd",
-    "params": [ { "vname": "code", "type": "Int32", "value": "3" } ]
-  },
+  "message": [
```

Comment:
Ideally this group should now be called "messages", but I don't know how big a change that is on the C++ side.

Reply:
Fixed this now. @kaikawaliu is fine with this change and he'll incorporate it in the core side also.